cli/README.md

```diff
@@ -1,6 +1,6 @@
-# redoc-cli
+# Redoc-cli
```

Comment:
Again, prolly not the ideal way to capitalize, given the usage elsewhere in lowercase anyways…

Reply:
This needs to be checked by the staff reviewers on the correct terminologies